src/Core/Managed/Net45/Extensibility/Implementation/External/UserContextData.cs

```diff
@@ -77,6 +77,15 @@ public string StoreRegion
         {
             get { return this.tags.GetTagValueOrNull(ContextTagKeys.Keys.UserStoreRegion); }
             set { this.tags.SetStringValueOrRemove(ContextTagKeys.Keys.UserStoreRegion, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the AuthUserId of an application-defined account associated with the user.
+        /// </summary>
+        public string AuthUserId

```

Comment:
This is the generated class.
I missed updating the public "UserContext" class. I've updated the PR with the additional change.  I named the property "AuthenticatedUserId" - does that sound good to you?